src/ui/view/manager.rs

```diff
@@ -336,25 +323,249 @@ impl ManagerSelectFeeView {
+                                .align_items(Align::Center),
+                        )))
+                        .align_items(Align::Center)
+                        .max_width(1000),
+                )
+                .align_x(Align::Center),
+            )
+            .push(footer.max_width(1000).align_items(Align::Center))
+            .align_items(Align::Center)
+            .spacing(20);
+
+        Container::new(Scrollable::new(&mut self.scroll).push(Container::new(col)))
+            .padding(20)
+            .style(ContainerBackgroundStyle)
+            .width(Length::Fill)
+            .height(Length::Fill)
+            .into()
+    }
+}
+
+fn spend_tx_with_feerate_view<'a, T: 'a>(
+    ctx: &Context,
+    inputs: &[model::Vault],
+    psbt: &Psbt,
+    feerate: &u32,
+) -> Container<'a, T> {
+    let mut total_fees = 0;
+    let mut col_input = Column::new()
+        .push(text::bold(text::simple("Inputs")))
+        .spacing(10);
+    for input in inputs {
+        total_fees += input.amount;
+        col_input = col_input.push(card::simple(Container::new(
+            Row::new()
+                .push(Container::new(text::small(&input.address.to_string())).width(Length::Fill))
+                .push(
+                    Container::new(text::bold(text::small(&format!(
+                        "{}",
+                        ctx.converter.converts(input.amount),
+                    ))))
+                    .width(Length::Shrink),
+                )
+                .spacing(5)
+                .align_items(Align::Center),
+        )));
+    }
+    let mut col_output = Column::new()
+        .push(text::bold(text::simple("Outputs")))
+        .spacing(10);
+    for output in &psbt.global.unsigned_tx.output {
+        total_fees -= output.value;
+        let addr = bitcoin::Address::from_script(&output.script_pubkey, ctx.network).unwrap();
+        col_output = col_output.push(card::simple(Container::new(
+            Row::new()
+                .push(Container::new(text::small(&addr.to_string())).width(Length::Fill))
+                .push(Container::new(
+                    text::bold(text::small(&format!(
+                        "{}",
+                        ctx.converter.converts(output.value)
+                    )))
+                    .width(Length::Shrink),
+                ))
+                .spacing(5)
+                .align_items(Align::Center),
+        )));
+    }
+    Container::new(
+        Column::new()
+            .push(
+                Container::new(text::bold(text::simple("Generated transaction")))
+                    .width(Length::Fill)
+                    .align_x(Align::Center),
+            )
+            .push(
+                Column::new()
+                    .push(
+                        Row::new()
+                            .push(text::simple("Feerate: "))
+                            .push(text::bold(text::simple(&format!("{} s/byte", feerate)))),
```

Comment:
nit: I would use `sats/vbyte`? What do you think?

Reply:
good idea